src/MusicalScore/Graphical/EngravingRules.ts

```diff
@@ -292,11 +294,13 @@ export class EngravingRules {
         this.wedgeLineWidth = 0.12;
         this.tupletLineWidth = 0.12;
         this.lyricUnderscoreLineWidth = 0.12;
+        this.lyricUnderscoreLineWidthVexflow = 1.3;
```

Comment:
as i see it now: can there again be the factor 10 (pixels per unit) missing? Looks like it...

Reply:
done.